routing/ROUTING_V1_HTTP.md

```diff
@@ -173,3 +185,9 @@ Specification: [ipfs/go-graphsync/blob/main/docs/architecture.md](https://github
 - `PieceCID`: the CID of the [piece](https://spec.filecoin.io/systems/filecoin_files/piece/#section-systems.filecoin_files.piece) within which the data is stored
 - `VerifiedDeal`: whether the deal corresponding to the data is verified
 - `FastRetrieval`: whether the provider claims there is an unsealed copy of the data available for fast retrieval
+
+## Known Routers
+
+This section contains a non-exhaustive list of known routers that may be supported by clients and servers.
+
+- "ipfs-public-dht" - The IPFS Public DHT corresponding to the libp2p protocol ID `/ipfs/kad/1.0.0`
```

Comment:
should we include / how should we name 'server's connected bitswap peers'?

Reply:
`ipfs-bitswap-peers` ?
I think it is ok to define some canonical names here, but the spec should be clear that it is at liberty of implementer to use other, opaque strings (for custom / future routing systems)